tests/clients/test_experiment.py

```diff
@@ -297,6 +317,11 @@ def test_metric_schema():
     assert data == METRIC
 
 
+def test_validation_metric_schema():
```

Comment:
There's no longer a reason to treat step differently on load. I suggest removing this test.